NOTES.md

```diff
@@ -0,0 +1,14 @@
+## 1. How do I set up my local environment to run the app locally?
```

Comment:
Question 1 and 2 seem very light on info; should we think about leaving them out for now?

It might be useful to mention the list of frameworks/libraries we use in our app in `README.md`, I think. Something like this:

```
    Google Sheets as database
    Next.js as app framework
    Chakra for UI framework
    Highcharts to build charts in report
    Google sheets to process form responses and calculate emissions
    Vercel as hosting platform
```